firefox-ios/Client/Frontend/Browser/Search/SearchViewController.swift

```diff
@@ -691,6 +702,28 @@ class SearchViewController: SiteTableViewController,
                                    _ indexPath: IndexPath) -> UITableViewCell {
         var cell = UITableViewCell()
         switch section {
+        case .trendingSearches:
+            if let trendingSearch = viewModel.trendingSearches[safe: indexPath.row] {
+                // TODO: FXIOS-13690 - Add one line cell configuration to be reusable.
```

Comment:
These cell configurations could be cleaner and reusable, but instead of refactoring in this PR, I created a new ticket to avoid this PR from having too many different contexts.